src/Elasticsearch.Net/Domain/RequestParameters/RequestParameters.Generated.cs

```diff
@@ -7133,6 +7167,66 @@ public class GetRoleRequestParameters : FluentRequestParameters<GetRoleRequestPa
 		
 	}
 	
+	///<summary>Request parameters descriptor for XpackSecurityGetRoleMapping
+	///<pre>
+	///https://www.elastic.co/guide/en/x-pack/master/security-api-role-mapping.html#security-api-get-role-mapping
+	///</pre>
+	///</summary>
+	public class GetRoleMappingRequestParameters : FluentRequestParameters<GetRoleMappingRequestParameters> 
```

Comment:
I think this should also be consistent with other Get* responses in Elasticsearch in returning a `{ "found": false }` body along with 404 response. Speaking with @tvernum, this behaviour is consistent across Get* responses in X-Pack Security, but going to open up an issue to collectively discuss these.